infrastructure/iac/iac.go

```diff
@@ -385,8 +409,56 @@ func (iac *Scanner) toAdditionalData(affectedFilePath string, issue iacIssue) sn
 		Issue:         issue.IacDescription.Issue,
 		Impact:        issue.IacDescription.Impact,
 		Resolve:       issue.IacDescription.Resolve,
-		Path:          issue.Path,
+		Path:          iacIssuePath,
 		References:    issue.References,
+	}, nil
+}
+
+func (iac *Scanner) parseIacIssuePath(path []interface{}) ([]string, error) {
```

Comment:
How about writing a few small unit tests for the function?

Reply:
how about using `[]any` type? We are using go 1.19, and personally, I like it better than `[]interface{}`

Reply:
Changed in https://github.com/snyk/snyk-ls/pull/269/commits/d53881ee92568432ace2f313971cb3a49ed4c491